ring/basic_lifecycler.go

```diff
@@ -53,6 +53,8 @@ type BasicLifecyclerConfig struct {
 	HeartbeatTimeout    time.Duration
 	TokensObservePeriod time.Duration
 	NumTokens           int
+	// HideTokens allows tokens to be hidden from e.g. the status page, for use in contexts which do not utilize tokens.
+	HideTokens bool
```

Comment:
nit: Maybe we can name this to indicate it's just for the status page? `HideTokens` at the top-level config sounds scary and behavior-changing.

Reply:
Good call - made the name more specific.